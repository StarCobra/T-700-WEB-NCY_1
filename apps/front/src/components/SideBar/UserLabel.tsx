import React from "react"
import profilPic from "../../assets/sideBar/profil.png"
export default function UserLabel() {
  return (
    <div className="userLocation">
      <div className="userProfilePicture">
        <img src={profilPic} />
      </div>

      <div className="usernameContainer">
        <div className="userName">Arnaud B</div>

        <div className="userMail">arnaud.bourgoin@gmail.com</div>
      </div>
    </div>
  )
}
