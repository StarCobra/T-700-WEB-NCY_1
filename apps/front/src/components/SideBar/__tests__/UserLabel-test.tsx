import React from "react";
import {act, create} from "react-test-renderer";
import UserLabel from "../UserLabel";

describe("Components > CryptoArray > CryptoArrayDisplay", () => {
    const tests = [
        {
            description: "generic",
            props: {},
        },
    ];
    tests.forEach((test) => {
        it(`should render the component ${test.description}`, async () => {
            let root;
            await act(async () => {
                root = create(<UserLabel {...test.props} />);
            });
            expect(root).toMatchSnapshot();
        });
    });
});
