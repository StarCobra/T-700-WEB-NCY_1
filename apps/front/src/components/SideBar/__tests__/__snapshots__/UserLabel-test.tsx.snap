// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Components > CryptoArray > CryptoArrayDisplay should render the component generic 1`] = `
<div
  className="userLocation"
>
  <div
    className="userProfilePicture"
  >
    <img
      src="default"
    />
  </div>
  <div
    className="usernameContainer"
  >
    <div
      className="userName"
    >
      Arnaud B
    </div>
    <div
      className="userMail"
    >
      arnaud.bourgoin@gmail.com
    </div>
  </div>
</div>
`;
